colandr/lib/extractors/locations.py

```diff
@@ -15,13 +15,6 @@
 class LocationExtractor:
     """Extract locations from document text using spaCy NER."""
 
-    def __init__(self, model: str = "en_core_web_md"):
-        """Initialize the extractor with the specified language model."""
-        self.nlp = textacy.load_spacy_lang(model)  # TODO: implement language detection
-        # Ensure we have sentence segmentation and named entity recognition
-        if not self.nlp.has_pipe("ner"):
-            raise ValueError(f"Model {model} doesn't have NER component")
```

Comment:
just checking, are you sure NER is included in the pipeline accessed via `process_texts_into_docs()`?